hibernate-reactive-core/src/test/java/org/hibernate/reactive/BasicTypesAndCallbacksTest.java

```diff
@@ -1,359 +0,0 @@
-									.thenCompose(v -> s3.refresh(basic))
-									.thenAccept(v -> context.assertEquals(basic.getString(), "Goodnight"))
-									.thenCompose(v -> {
-										basik.version = basic.version;
-										return s3.merge(basik);
-									})
-									.thenAccept( b -> context.assertEquals( b.string, "Hello World") )
-									.thenCompose(v -> s3.remove(basic))
-									.thenAccept(v -> context.assertTrue( !basic.postRemoved && basic.preRemoved ) )
-									.thenCompose(v -> s3.flush())
-									.thenAccept(v -> context.assertTrue( basic.postRemoved && basic.preRemoved ) );
-						}))
-				.thenCompose(v -> openSession())
-				.thenCompose(s4 ->
-						s4.find( Basic.class, basik.getId() )
-							.thenAccept(context::assertNull))
-		);
-	}
-
-	enum Cover { hard, soft }
-
-	@Embeddable
-	static class Embed {
-		String one;
-		String two;
-
-		public Embed(String one, String two) {
-			this.one = one;
-			this.two = two;
-		}
-
-		Embed() {}
-
-		public String getOne() {
-			return one;
-		}
-
-		public void setOne(String one) {
-			this.one = one;
-		}
-
-		public String getTwo() {
-			return two;
-		}
-
-		public void setTwo(String two) {
-			this.two = two;
-		}
-
-		@Override
-		public boolean equals(Object o) {
-			if (this == o) return true;
-			if (o == null || getClass() != o.getClass()) return false;
-			Embed embed = (Embed) o;
-			return Objects.equals(one, embed.one) &&
-					Objects.equals(two, embed.two);
-		}
-
-		@Override
-		public int hashCode() {
-			return Objects.hash(one, two);
-		}
-	}
-
-	@Entity(name="Basic") @Table(name="Basic")
-	public static class Basic {
-
-		@Id @GeneratedValue Integer id;
-		@Version Integer version;
-		String string;
-
-		boolean primitiveBoolean;
-		int primitiveInt;
-		long primitiveLong;
-		float primitiveFloat;
-		double primitiveDouble;
-		byte primitiveByte;
-
-		Boolean nullBoolean;
-		Integer nullInt;
-		Long nullLong;
-		Float nullFloat;
-		Double nullDouble;
-		Byte nullByte;
-
-		@Column(name="dessimal")
-		BigDecimal decimal;
-		@Column(name="inteja")
-		BigInteger integer;
-		byte[] bytes;
-		Cover cover;
-		@javax.persistence.Basic
-		Serializable thing;
-		TimeZone timeZone;
-		@Temporal(TemporalType.DATE)
-		private Date date;
-		@Column(name="localdayte")
-		private LocalDate localDate;
-		@Column(name="localdaytetyme")
-		private LocalDateTime localDateTime;
-		@Column(name="localtyme")
-		private LocalTime localTime;
-		@Convert(converter = BinInteger.class)
-		private BigInteger binInteger;
-
-		@ManyToOne(fetch = FetchType.LAZY)
-		Basic parent;
-
-		Embed embed;
-
-		@Lob @Column(length = 100_000) protected byte[] pic;
-		@Lob @Column(length = 100_000) protected String book;
-
-		@Transient boolean prePersisted;
-		@Transient boolean postPersisted;
-		@Transient boolean preUpdated;
-		@Transient boolean postUpdated;
-		@Transient boolean postRemoved;
-		@Transient boolean preRemoved;
-		@Transient boolean loaded;
```

Comment:
I don't see what happened to the tests for JPA callbacks?

Reply:
They are all in: BasicTypesAndCallbacksForAllDBsTest

Reply:
https://github.com/hibernate/hibernate-reactive/blob/abb3ecd984df9847a7b8c52ac897248f24c25039/hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/BasicTypesAndCallbacksForAllDBsTest.java#L279